modules/ipi-install-node-requirements.adoc

```diff
@@ -59,5 +59,5 @@ When starting the installation from virtual media such as an ISO image, delete a
 +
 [NOTE]
 ====
-Red Hat does not support Secure Boot with self-generated keys.
+Red Hat supports systems where the self-generated keys are enrolled. Red Hat does not support managing self-generated keys for Secure Boot.
```

Comment:
maybe we want to keep it short:
Red Hat does not support managing of self-generated (or other) keys for secure boot

or if want to be precise:
Red Hat supports secure boot on systems where the original Microsoft UEFI CA was not removed.

(and not mention anything about managing keys at all)